docs/sphinx/source/whatsnew/v0.4.6.rst

```diff
@@ -0,0 +1,30 @@
+.. _whatsnew_0460:
+
+v0.4.6 ()
+---------
+
+
+Bug fixes
+~~~~~~~~~
+
+* Method of multi-inheritance has changed to make it possible to use kwargs in
+  the parent class (:issue:`330`)
```

Comment:
Would also be good to update the PR name to something like "allow kwargs in parent class of LocalizedPVSystem and LocalizedSingleAxisTracker"